vignettes/design_principle.Rmd

```diff
@@ -302,15 +302,16 @@ These surrogate functions play crucial roles in the data analysis and cleaning w
 The modules and the surrogate functions depend on the following packages:
 
 [{numberize}](https://github.com/epiverse-trace/numberize) used for the conversion of number from character to numeric, [{dplyr}](https://CRAN.R-project.org/package=dplyr) used in many ways including filtering, column creation, data summary, etc, [{magrittr}](https://CRAN.R-project.org/package=magrittr) used here for its `%>%` operator,
-[{linelist}](https://CRAN.R-project.org/package=linelist) used to perform some operations on linelist-type input objects, [{matchmaker}](https://CRAN.R-project.org/package=matchmaker) utilized to perform the dictionary-based cleaning, [{lubridate}](https://CRAN.R-project.org/package=dplyr) used to create, handle, and manipulate objects of type Date, [{reactable}](https://CRAN.R-project.org/package=reactable) mainly used here to customize the data cleaning report, [{cli}](https://CRAN.R-project.org/package=cli) utilized to reformat all the messages in the package (from version 1.1.0 and above),
+[{linelist}](https://CRAN.R-project.org/package=linelist) used to perform some operations on linelist-type input objects, [{matchmaker}](https://CRAN.R-project.org/package=matchmaker) utilized to perform the dictionary-based cleaning, [{lubridate}](https://CRAN.R-project.org/package=dplyr) used to create, handle, and manipulate objects of type Date, [{reactable}](https://CRAN.R-project.org/package=reactable) mainly used here to customize the data cleaning report,
+[{cli}](https://CRAN.R-project.org/package=cli) utilized to reformat all the messages in the package (from version 1.1.0 and above),
 [{rlang}](https://CRAN.R-project.org/package=rlang) used here for its `.data` functionality to retrieve data-variables from the data frame, [{readr}](https://CRAN.R-project.org/package=readr) used to import data, [{janitor}](https://CRAN.R-project.org/package=janitor) used to standardize column names in `standardize_column_names()`,
 [{tibble}](https://CRAN.R-project.org/package=tibble) used to work tibbles.
 [{naniar}](https://CRAN.R-project.org/package=tibble) used to for its predefined set of common missing value strings.
 
 The functions will require other packages that are needed in the package development process including:
 
 [{checkmate}](https://CRAN.R-project.org/package=checkmate), [{kableExtra}](https://CRAN.R-project.org/package=kableExtra), [{bookdown}](https://CRAN.R-project.org/package=bookdown), [{rmarkdown}](https://CRAN.R-project.org/package=rmarkdown), [{testthat}](https://CRAN.R-project.org/package=testthat) (\>= 3.0.0), [{knitr}](https://CRAN.R-project.org/package=knitr), [{lintr}](https://CRAN.R-project.org/package=lintr),
-[{utils}](https://CRAN.R-project.org/package=utils),
+[{utils}](https://CRAN.R-project.org/package=R.utils),
```

Comment:
utils & R.utils are two different packages. 

```suggestion
{utils}
```

Reply:
Thanks for the catch.